src/client/modules/ExportWins/Form/WinDetailsStep.jsx

```diff
@@ -1,21 +1,225 @@
 import React from 'react'
+import { Details, ListItem, UnorderedList } from 'govuk-react'
+import { H3 } from '@govuk-react/heading'
 import styled from 'styled-components'
 
+import ResourceOptionsField from '../../../components/Form/elements/ResourceOptionsField'
 import { useFormContext } from '../../../../client/components/Form/hooks'
-import { Step, FieldInput } from '../../../components'
-import { steps } from './constants'
+import CountriesResource from '../../../components/Resource/Countries'
+import { BLACK, WHITE } from '../../../../client/utils/colours'
+import { SectorResource } from '../../../components/Resource'
+import { OPTION_YES } from '../../../../common/constants'
+import { WinTypeValues } from './WinTypeValues'
+import { StyledHintParagraph } from './styled'
+import {
+  Step,
+  FieldDate,
+  FieldInput,
+  FieldTextarea,
+  FieldTypeahead,
+  FieldCheckboxes,
+} from '../../../components'
+import {
+  steps,
+  winTypes,
+  winTypeOptions,
+  goodsServicesOptions,
+} from './constants'
+import {
+  formatValue,
+  getTwelveMonthsAgo,
+  sumAllWinTypeYearlyValues,
+} from './utils'
 
-const StyledFieldInput = styled(FieldInput)({
-  display: 'none',
+const MAX_WORDS = 100
+
+const StyledDetails = styled(Details)({
+  margin: 0,
+})
+
+const StyledExportTotal = styled('p')({
+  padding: 10,
+  color: WHITE,
+  backgroundColor: BLACK,
 })
 
 const WinDetailsStep = () => {
-  // eslint-disable-next-line no-unused-vars
   const { values } = useFormContext()
+  const twelveMonthsAgo = getTwelveMonthsAgo()
+  const month = twelveMonthsAgo.getMonth() + 1
+  const year = twelveMonthsAgo.getFullYear()
+
   return (
     <Step name={steps.WIN_DETAILS}>
-      <h1>Win details</h1>
-      <StyledFieldInput name="hidden" type="text" />
+      <H3 data-test="step-heading">Win details</H3>
+      <StyledHintParagraph data-test="hint">
+        The customer will be asked to confirm this infomation.
```

Comment:
```suggestion
        The customer will be asked to confirm this information.
```
There's a small typo